services/core-web/src/components/mine/Projects/MajorMineApplicationTab.tsx

```diff
@@ -38,6 +38,8 @@ const MajorMineApplicationTab: FC = () => {
   const mineDocuments = useSelector(getMineDocuments);
   const dropdownMajorMineAppStatusCodes = useSelector(getDropdownMajorMinesApplicationStatusCodes);
   const archivedDocuments = mineDocuments?.map((doc) => new MajorMineApplicationDocument(doc))
+  const authorizations = project?.project_summary?.authorizations;
+  const hasEmaApp = authorizations?.some(auth => auth.ams_tracking_number);
```

Comment:
I think there's a few instances here of `hasEmaApp` in different files. Unless you are only looking at submitted to AMS applications, definitely use the authorization type instead.